### PR TITLE
Quick feature: Co-insured for accident

### PR DESCRIPTION
**Pull request by rasmusguterstam**
## What?
- Add option to add co-insured to a claim

## Optional screenshots

https://user-images.githubusercontent.com/51323202/136528251-72c1fd15-29f3-4ff4-aa47-9c3c2a48df9e.mov

## Optional checklist
- [x] Updated `src/changelog.ts`
- [ ] Codescouted
- [ ] Unit tests written
- [x] Tested locally

### CODE REVIEW

File: src/changelog.ts [hunk 1]
```diff
@@ -11,6 +11,12 @@ export interface Change {
 }
 
 export const changelog: ReadonlyArray<Change> = [
+  {
+    date: '2021-10-08',
+    change:
+      'Added keyboard navigation between Claim sections using Control+Number',
```
Comment (vonElfvin) [inline src/changelog.ts:17]: You wut mate

File: src/features/claims/claim-details/components/ClaimInformation.tsx [hunk 2]
```diff
@@ -105,6 +110,9 @@ export const ClaimInformation: React.FC<{
   memberId: string
   focus: boolean
 }> = ({ claimId, memberId, focus }) => {
+  const [creatingCoInsured, setCreatingCoInsured] = useState(false)
```
Comment (vonElfvin) [inline src/features/claims/claim-details/components/ClaimInformation.tsx:113]: An idea maybe? Then we could just use one variable after
```suggestion
  const [hasCoInsured, setHasCoInsured] = useState(!!coInsured)
```

File: src/features/claims/claim-details/components/ClaimInformation.tsx [hunk 3]
```diff
@@ -232,6 +241,30 @@ export const ClaimInformation: React.FC<{
             ]}
           />
         </SelectWrapper>
+        <SelectWrapper>
+          <Label>Co-insured Claim</Label>
+          <SemanticDropdown
+            value={creatingCoInsured || coInsured ? 'True' : 'False'}
```
Comment (vonElfvin) [inline src/features/claims/claim-details/components/ClaimInformation.tsx:247]:
```suggestion
            value={hasCoInsured ? 'True' : 'False'}
```

**Review by vonElfvin — APPROVED** [review]
🔥

File: src/changelog.ts (hunk 1 above)
Comment (rasmusguterstam) [reply src/changelog.ts:17]: Loool

**Pull request merged**